### PR TITLE
Release (minor): 0.34.0 (Smart Transfer Protocol: breaking changes)

### PR DESCRIPTION
**Pull request by s373r**
### Changed
- Upgrade kamu-cli version to `0.198.2`
  - ReBAC: in-memory & SQLite components
  - Smart Transfer Protocol: breaking changes

Based on: 
- https://github.com/kamu-data/kamu-cli/pull/761
- https://github.com/kamu-data/kamu-cli/pull/782

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Changed
+- Upgrade kamu-cli version to 0.XXX.X (ReBAC, TODO)
```
Comment (s373r) [inline CHANGELOG.md:10]: TODO: update the text to the actual text when PR is merged
- https://github.com/kamu-data/kamu-cli/pull/761

File: Cargo.toml [hunk 2]
```diff
@@ -16,44 +16,47 @@ resolver = "2"
...
-database-common = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-database-common-macros = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-http-common = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-internal-error = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-messaging-outbox = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-random-names = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-time-source = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
+container-runtime = { git = "https://github.com/kamu-data/kamu-cli", branch = "feature/727-Private-Datasets-Update-use-case-Creating-a-dataset", default-features = false }
```
Comment (s373r) [inline Cargo.toml:19]: TODO: set the correct version when the PR is merged
- https://github.com/kamu-data/kamu-cli/pull/761

**Review by s373r — COMMENTED** [review]
Add TODOs

File: CHANGELOG.md (hunk 1 above)
Comment (s373r) [reply CHANGELOG.md]: Done

File: Cargo.toml (hunk 2 above)
Comment (s373r) [reply Cargo.toml]: Done

**Review by rmn-boiko — APPROVED** [review]

**Pull request merged**